.github/CODEOWNERS

```diff
@@ -3,62 +3,62 @@
 # Please define less specific codeowner paths before more specific codeowner paths in order for the more specific rule to have priority
 
 # Root
-* @smartcontractkit/foundations
+* @smartcontractkit/core
```

Comment:
Since this root definition is present, do we even need to list core under all of these other paths?

Reply:
IIUC the root is overridden by the more specific filters, so we have to list core again if we want it to be included.